tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -112,6 +112,17 @@ void toPointCloud2(const TangoPointCloud& tango_point_cloud,
   }
   point_cloud->header.stamp.fromSec((tango_point_cloud.timestamp + time_offset) / 1e3);
 }
+// Transform a point from frame 1 to frame2.
+// @param frame1_p point expressed in frame1
+// @param frame1_T_frame2_message transform message from frame1 to frame2.
+// @param frame2_p output point expressed in frame2.
+void transformPoint(const tf::Vector3& frame1_p,
+                    const geometry_msgs::Transform& frame1_T_frame2_message,
```

Comment:
Would be better if the function directly takes tf::Transform as input. This avoids converting the message type to a transform type for every point in the point cloud.